modules/apps/frontend-data-set/frontend-data-set-impl/src/test/java/com/liferay/frontend/data/set/internal/serializer/SystemFDSSerializerTest.java

```diff
@@ -1186,6 +1189,24 @@ public String getDescription() {
 				return DESCRIPTIONS[0];
 			}
 
+			@Override
+			public FDSCardSchema getFDSCardSchema(Locale locale) {
+				FDSCardSchemaBuilder fdsCardSchemaBuilder =
+					new FDSCardSchemaBuilderImpl();
+
+				return fdsCardSchemaBuilder.add(
+					"success", "Approved"
```

Comment:
As spoken in person, @Mikellorza please change all strings here by their corresponding constans, asserting them later on

Reply:
@dsanz constants added